.buildkite/pipeline-master.yml

```diff
@@ -37,18 +37,26 @@ steps:
     command: "./scripts/buildkite/fossa.sh"
 
   - label: ":golang: unit test"
-    agents:
-      queue: "workers"
-    commands:
-      - "CASSANDRA_HOST=cassandra make install-schema && make cover_profile" # make install-schema is needed for a server startup test. See main_test.go
-      - "./scripts/buildkite/gen_coverage_metadata.sh .build/coverage/metadata.txt"
     artifact_paths:
       - ".build/coverage/*.out"
       - ".build/coverage/metadata.txt"
     retry:
       automatic:
         limit: 1
     plugins:
+      - kubernetes:
```

Comment:
I was expecting this change to cause `unit test` to time out because it was consistently failing during initial migration. I guess some recent changes improved the matching folder's test duration

Reply:
I made some improvements on the matching tests last Friday.